### ISSUE TITLE
Schema errors at SwaggerHub

### ISSUE THREAD

**Issue by tormi** [acquia/reservoir#17]
Tried to use OpenAPI specification generated by reservoir at swaggerhub, got a bunch of schema errors: https://app.swaggerhub.com/apis/tormi/reservoir-json_api/1.0.0 This is probably also from upstream.

**Comment by wimleers** [comment]
Thanks for reporting this! Paging @tedbow, https://www.drupal.org/project/openapi maintainer.

**Comment by tedbow** [comment]
@tormi yep. thanks for reporting this. Please an issue at https://www.drupal.org/project/openapi

I will also try the export their. The OpenAPI is under active development.

**Comment by tormi** [comment]
Done: https://www.drupal.org/node/2888294
